tests/test.sh

```diff
@@ -1,3 +1,61 @@
-#!/bin/sh
+#!/bin/bash
 
-echo "Not implemented, so it passed"
+cd tests
+
+source ./testutil.sh
+
+message "Running synbiohub test suite."
+message "Cleaning old test containers if they exist"
+
+bash ./testcleanup.sh
+
+
+
+# Clone the necessary repositories
+message "pulling mehersam/SBOLTestRunner"
+if cd SBOLTestRunner; then
+    git pull;
+    cd ..;
+else
+    git clone --recurse-submodules https://github.com/mehersam/SBOLTestRunner;
+fi
+
+
+
+#message pulling mhersam/SBOLTestRunner
+#if cd SynBioHubRunner
+#git clone --recurse-submodules https://github.com/mehersam/SynBioHubRunner
+
+message "pulling synbiohub/synbiohub-docker"
+if cd synbiohub-docker; then
+    git pull;
+    cd ..;
+else
+    # clone the synbiohub docker compose file in order to run docker containers
+    git clone https://github.com/synbiohub/synbiohub-docker;
+fi
+
+message "Building SBOLTestRunner"
+cd SBOLTestRunner
+mvn package
+cd ..
+
+message "Starting SynBioHub from Containers"
+docker-compose -f ./synbiohub-docker/docker-compose.yml -p testproject up &
+while [[ "$(docker logs --tail 1 testproject_synbiohub_1)" != "Resuming 0 job(s)" ]]
+do
+    sleep 5
+done
+
+message "Started successfully"
+
+# wait for logs of other containers
+sleep 30s
```

Comment:
If SynBioHub is healthy, the other containers should be too. The only exception *might* be ElasticSearch, but you can safely ignore that for now. If we want to test SBOLExplorer we will have to create a healthcheck for ElasticSearch

Reply:
Thank you for all the help.